Packages.props

```diff
@@ -54,6 +54,7 @@
     <PackageReference Update="Microsoft.SqlServer.XEvent.XELite" Version="2023.1.30.3" />
     <PackageReference Update="SkiaSharp" Version="2.88.6" />
     <PackageReference Update="SkiaSharp.NativeAssets.Linux.NoDependencies" Version="2.88.6" Condition="$([MSBuild]::IsOsPlatform('Linux'))" />
+    <PackageReference Update="TextCopy" Version="6.2.1" />
```

Comment:
I did a quick check and found that it's tricky to access clipboard on Linux, I believe this package doesn't work with Ubuntu 22.04 without additional package. See https://github.com/CopyText/TextCopy/issues/733

Can you give this package a test on all platforms and see
1) does it work?
2) is it worth it to add this package? or do we want to write something of our own (win should be easy, for mac see https://stackoverflow.com/questions/51909120/how-to-get-the-clipboard-text-on-osx-using-dllimport-with-c)

If we believe it makes sense to use this package, then you might also want to add the nuget page / repo page / license info of this package to the PR description.

Reply:
@caohai, you're right, this isn't going to work. TextCopy version 6.2.1 was used in [STS PR #2096](https://github.com/microsoft/sqltoolsservice/pull/2096/files#diff-202f452c0bbb139bd6a395d212e8b6cfb7909dc37a19a6b600ef1adf98c7662dR50) and it broke copying on Linux. A follow up PR was then made to copy from ADS instead of copying from STS here: https://github.com/microsoft/azuredatastudio/pull/24341.

I'll continue to look into how to fix this for Linux.

Reply:
@lewis-sanchez do you know why with @alanrenmsft PR (https://github.com/microsoft/sqltoolsservice/pull/2096/) we're still seeing slow copy experience?  That PR seems to be doing what I thought this PR was going to do, and the recent bug report isn't for Linux (https://github.com/microsoft/azuredatastudio/issues/25632).